src/content/guides/getting-started.md

```diff
@@ -98,13 +124,15 @@ __project__
     |- index.js
 ```
 
-To bundle the `lodash` dependency with `index.js`, we'll need to install the library locally...
+To bundle the `lodash` dependency with `index.js`, we'll need to install the library locally:
 
 ``` bash
-npm install --save-dev lodash
+npm install --save lodash
 ```
 
-and then import it in our script...
+T> As a rule of thumb: if you're installing a package that will be bundled into your rpdocution code, then you should use `npm install --save`. If you're installing a package to help development (e.g. a linter, testing libraries, etc.) then you should use `npm install --save-dev`. More information can be found in the [npm documentation](https://docs.npmjs.com/cli/install).
```

Comment:
We could shorten this to:

_When installing a package for production or runtime bundles we use the `--save` flag. On the other hand, the `--save-dev` flag is for packages related to development tasks like linters or testing libraries. More information can be found in the [npm documentation](https://docs.npmjs.com/cli/install)._

What do you think?

Reply:
Hmm, apart from my typo (lol), I kind of like to be a bit explicit about it (for new users) and clearly communicate that libs used in the bundle fall under deps.

I don't think it can hurt in this case to be a bit verbose. I'd rather leave it this way, is that okay with you as well?

Reply:
Yeah, it's ok. It's jsut me trying to keep tips as short as possible to avoid big color blocks and drive away the atention on the main content. How about:

_When installing a package that will be bundled into your production or runtime code, you should use `npm install --save`. If you're installing a package for development purposes (e.g. a linter, testing libraries, etc.) then you should use `npm install --save-dev`. More information can be found in the [npm documentation](https://docs.npmjs.com/cli/install)._

Just some gardening around the words 😊

Reply:
`s/rpdocution/production`